### PR TITLE
Adds in cell management console

### PR DESCRIPTION
**Pull request by AffectedArc07**
**What does this PR do:**
This PR adds in a new console for the brig, which allows you to view the status of all current brig cells. It looks like this.
image

When clicked, it will show this UI:
image
This has a list of all cell timers in the world, and their status. Status includes whether they are in use or not, and if they are, who is in there, who put them in there, what crimes they committed, how long they were brigged for, how long is left on their sentence, and a button to release them. 

**Images of sprite/map changes (IF APPLICABLE):**
New sprite (Thanks `Alonefromhell`:
image

#Map edits:
Wardens office:
image

Front Desk:
image

HoS office:
image

Few ideas for things, which will be added if asked:
Maybe an emag act which forces open all cells?
Maybe a log of previous brigs on a console instead of the sheets of paper?

**Changelog:**
:cl: AffectedArc07 and Alonefromhell
add: Cell management consoles: See the status of all brig cells without having to leave your office
imageadd: New screen sprite for cell management console
/:cl:

# Mapmerge
I think its just straight up broken. I did it on my previous PR and it didnt work, and I did it on this PR and it didnt work. Its not computer-related as I am using a different PC here than I used to make my previous PR. The log is below if it makes any sense
```
File to use: 0
Loading map cyberiad.dmm.backup
Loading tiles 8992
Loading map cyberiad.dmm
Loading tiles 8992
Loading levels
New map part from (1,1,1)
We have 8992 different tiles
Done
```

### CODE REVIEW

**Comment by farie82** [comment]
Small thing at first. Moving the HoS locker makes it easier for antags to steal his shit.

**Comment by AffectedArc07** [comment]
@farie82 Correct me if I am wrong, but the HoS usually has the gun on him. And if its in the locker, its just a few tiles closer to space than before. I *personally* dont see an issue here but again correct me if I am wrong

**Comment by datlo** [comment]
Perhaps add a read-only version of the console to IAA/Magistrate office/Bridge?
Could replace prisonner management consoles, I've never seen those used outside of specific situations where the ones in the brig would do the job.

This looks great tho, 👍

**Comment by farie82** [comment]
@AffectedArc07 it's in the locker at round start. It's a minor thing. Otherwise the PR is good.

**Comment by AffectedArc07** [comment]
@datlo I left those intact because in the 1 in a billion chance someone uses implants I want it to still work. I made a new console that used nanoUI because I wanted something that live updates. I /might/ make a read only version, depends where this PR goes. I may also replace the prisoner console in the bridge with this one, as the only place that really needs a prisoner console imo is the warden and HoS office.

**Comment by AffectedArc07** [comment]
Note to self: I cant click

File: code/game/machinery/computer/brigcells.dm [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        ui = new(user, src, ui_key, "brig_cells.tmpl", "Brig Cell Management", 1000, 400)
+        ui.open()
+        ui.set_auto_update(1)
+
+/obj/machinery/computer/brigcells/ui_data(mob/user, ui_key = "main", datum/topic_state/state = default_state)
+    var/data[0]
+    var/list/timers = list()
+    /*"Detainee [logname] has been incarcerated for [seconds_to_time(timetoset / 10)] for the charges of, '[logcharges]'. \
+	Arresting Officer: [usr.name].[R ? "" : " Detainee record not found, manual record update required."]"*/
+    for(var/obj/machinery/door_timer/T in world)
```
Comment (Fox-McCloud) [inline code/game/machinery/computer/brigcells.dm]: please make door_timers be in their own list.

`world` searches are disgusting.

File: code/game/machinery/computer/buildandrepair.dm [hunk 2]
```diff
@@ -219,6 +219,9 @@
 /obj/item/circuitboard/prisoner
 	name = "Circuit board (Prisoner Management)"
 	build_path = /obj/machinery/computer/prisoner
+/obj/item/circuitboard/brigcells
```
Comment (Fox-McCloud) [inline code/game/machinery/computer/buildandrepair.dm]: spacing between this and the one above it.

Comment (AffectedArc07) [reply code/game/machinery/computer/buildandrepair.dm]: None of these have spacing and spacing just one of them will look wierd

**Comment by Landerlow** [comment]
This is brilliant. Merge ASAP.

File: code/game/machinery/computer/buildandrepair.dm (hunk 2 above)
Comment (taukausanake) [reply code/game/machinery/computer/buildandrepair.dm]: I'd probably put spaces between all of them just because it's an eyesore

**Comment by Quantum-M** [comment]
Why not combine it with the console that tracks tracking implants? Since, I think, it prints out cell times and it would take less space.

**Comment by variableundefined** [comment]
@AffectedArc07 Did you rebase / merge changes from master properly? I see your commit is dirty and includes some recent PRs.

**Comment by AffectedArc07** [comment]
I think I had conflicts and tried to fix them and broke my tree. Would I try squashing to fix this?

**Comment by AffectedArc07** [comment]
Re-creating this because its an absolute fucking mess

**Comment by AffectedArc07** [comment]
I am gonna reopen this when #10945 gets merged because this had map conflicts and why fix the map if its gonna change again

**Pull request closed without merging**